tests/sentry/api/endpoints/test_project_rules_configuration.py

```diff
@@ -213,3 +214,36 @@ def test_high_priority_issue_condition_feature(self):
                 "sentry.rules.conditions.high_priority_issue.ExistingHighPriorityIssueCondition"
                 in [filter["id"] for filter in response.data["conditions"]]
             )
+
+    def test_has_in_feature(self):
+        with self.feature({"organizations:issues-alerts-is-in": False}):
```

Comment:
nit: `issue-alert-in-filter` might be a better name for the feature flag. the first time reading it i was like "what's it in?" 😅

Reply:
ah yea - since the flag is already merged in options automator i'll keep as is